.github/workflows/jacoco.yml

```diff
@@ -0,0 +1,51 @@
+name: Coverage
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Set up JDK 11 for Autogen
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 11
+          cache: 'maven'
```

Comment:
no need to use `cache: 'maven'` because you use cache@v4

Reply:
Got it. Will delete `cache: 'maven' ` on next commit.